src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ParameterAssignmentCheck.java

```diff
@@ -50,6 +50,30 @@
  * &lt;module name=&quot;ParameterAssignment&quot;/&gt;
  * </pre>
  * <p>
+ * Example :
+ * </p>
+ * <pre>
+ * class MyClass {
+ *     int method(int parameter) {
+ *         if (parameter &lt;= 0 ) {
+ *             throw new IllegalArgumentException("A positive value is expected");
+ *         }
+ *         parameter -= 2;  // violation
+ *         return parameter;
+ *     }
```

Comment:
please make one more example with stupid case when it is just an assignment to parameter (not -=)

Reply:
Done